ansible_ai_connect/users/templates/users/home.html

```diff
@@ -129,7 +129,7 @@ <h1 class="pf-c-title pf-m-lg">{{ project_name }}</h1>
                   {% if deployment_mode == 'saas' and user.is_authenticated and user.rh_user_is_org_admin %}
                   <a class="pf-l-level__item" href="/console"><span class="fas fa-solid fa-cog"></span> Admin Portal</a>
                   {% endif %}
-                  {% if chatbot_enabled and deployment_mode == 'saas' and user.is_authenticated and user.rh_employee %}
+                  {% if chatbot_enabled and deployment_mode == 'saas' and user.is_authenticated and rh_employee_or_test_user %}
```

Comment:
You also need to adjust https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/users/templates/users/trial.html#L134-L136